src/toil/jobStores/abstractJobStore.py

```diff
@@ -114,6 +158,81 @@ def _checkJobStoreCreation(create, exists, jobStoreString):
             raise JobStoreCreationException("The job store '%s' does not exist, so there "
                                             "is nothing to restart." % jobStoreString)
 
+    def importFile(self, srcUrl):
+        """
+        Imports the file pointed at by sourceUrl into jobStore. The jobStoreFileId of the new
+        file is returned.
+
+        :param str sourceUrl: URL that points to either a local file or to a file in a supported
+                jobStore e.g. 's3' for AWS.
+        :return The jobStoreFileId of imported file.
+        """
+        with self.writeFileStream() as (writable, jobStoreFileID):
+            findJobStoreForUrl(srcUrl).writeFromUrl(srcUrl, writable)
+            return jobStoreFileID
+
+    def exportFile(self, jobStoreFileID, dstUrl):
+        """
+        Exports file to destination pointed at by the destUrl.
+
+        :param str jobStoreFileID: The id of the file in the jobStore that should be exported.
+        :param str destUrl: URL that points to either a local file or to a file in a supported
+                jobStore e.g. 's3' for AWS.
```

Comment:
same as above... I did this in import file, should I have two copies of it for both methods?